R/make-request.R

```diff
@@ -0,0 +1,38 @@
+#' Make a Google API request
+#'
+#' Intended primarily for internal use in client packages that provide
+#' high-level wrappers for users. `make_request()` does very little: calls an
+#' HTTP method, only adding a user agent. Typically the input is created with
+#' [`build_request()`] and the output is processed with [`process_response()`].
+#'
+#' @param x List. Holding the components for an HTTP request, presumably created
+#'   with [`generate_request()`] or [build_request()]. Must contain the a
```

Comment:
wildly nitpicky: missing backticks around `build_request()`

Reply:
OK I sorted out a few backtick and link issues.